include/socgraph.php

```diff
@@ -200,20 +203,20 @@ function poco_load_worker($cid, $uid, $zcid, $url) {
  *  4: ...
  *
  */
-function sanitized_gcontact(&$gcontact) {
+function sanitize_gcontact(&$gcontact) {
```

Comment:
Can you please remove the reference operator and add a `@throw` tag to indicate this function throws `Exception`?